### DISCUSSION TITLE
How can I get same image from #1368

### DISCUSSION THREAD

**Discussion by Jaehwa-Noh** [android/nowinandroid#1370] [Q&A] [answered]
In this #1368 pull request has this image.
![image](https://github.com/android/nowinandroid/assets/48680511/e3896e20-7bd6-449c-8e68-b8336390f3bc)

I want to know how can I get that image.
Is this from Android Studio?
What tools to be used get the image?

@mlykotom Would you help me?

**Comment by mlykotom** [comment, marked as answer]
This is a screenshot of a system trace in https://perfetto.dev/. You record a system trace (in Android Studio, from terminal, or with Macrobenchmark) and then just open it in https://ui.perfetto.dev/

We actually made a video in the past about how to think about performance in compose, and it also includes how to get the traces and analyze it in Perfetto --> https://www.youtube.com/watch?v=Z96wfbID_Yc

**Reply by Jaehwa-Noh** [reply]
I will see the video, and check. Thank you.
